### ISSUE TITLE
Display genres in Artist and Album views

### ISSUE THREAD

**Issue by n-peugnet** [dkanada/gelli#104]
I would like to be able to see the genres of Artists and Albums as it is already possible in the webapp.

**Comment by grravity** [comment]
If you're suggesting the ability to see the genre of an album or artist, then click on the genre to see more, I +1 this.

**Comment by dkanada** [comment]
Duplicate of #67 and definitely planned for the future.

**Comment by n-peugnet** [comment]
I don't believe these are duplicates. Mine was about displaying the genre on album and artist pages. For instance in current web you can see the `Electronic` genre (and click on it), whereas on Gelli this information is nowhere to be found:

### Web
![2021-05-06-122256_1273x506_scrot](https://user-images.githubusercontent.com/23519418/117283623-41df9100-ae66-11eb-88d8-ea00d899cc66.png)

### Gelli

**Comment by dkanada** [comment]
You're right, I had misunderstood your request! I'll leave this open until it gets implemented.
